src/Avatar.js

```diff
@@ -1,31 +1,28 @@
-import React from 'react'
 import PropTypes from 'prop-types'
-import styled from 'react-emotion'
-import {themeGet} from 'styled-system'
-import {withSystemProps} from './system-props'
-
-const Avatar = styled(AvatarImage)`
-  display: inline-block;
-  overflow: hidden; // Ensure page layout in Firefox should images fail to load
-  line-height: ${themeGet('lineHeights.condensedUltra', 1)};
-  vertical-align: middle;
-  ${borderRadius};
-`
+import styled from 'styled-components'
+import {themeGet, space, color} from 'styled-system'
+import theme from './theme'
 
 function borderRadius({size}) {
   return {
     borderRadius: size <= 24 ? '2px' : '3px'
   }
 }
 
-function AvatarImage({alt, size, ...rest}) {
-  return <img alt={alt} width={size} height={size} {...rest} />
-}
-
-// styled() changes this
-Avatar.displayName = 'Avatar'
+const Avatar = styled.img.attrs(props => ({
+  height: props.size,
+  width: props.size
+}))`
+  display: inline-block;
+  overflow: hidden; // Ensure page layout in Firefox should images fail to load
+  line-height: ${themeGet('lineHeights.condensedUltra', 1)};
+  vertical-align: middle;
+  ${borderRadius};
+  ${space} ${color};
```

Comment:
Looks like `color` wasn't previously here and isn't needed so I need to remove this